### ISSUE TITLE
oas3-valid-oas-content-example value type

### ISSUE THREAD

**Issue by DerManoMann** [stoplightio/spectral#1780]
Something somewhere between a question and a bug report...

`spectral` reports an error for `oas3-valid-oas-content-example` where the value of a content example is not an object.

Given this fragment in the spec:
```yaml
      responses:
        '200':
          description: OK
          content:
            application/json:
              schema:
                oneOf:
                  - { $ref: '#/components/schemas/Result' }
                  - { type: boolean }
              examples:
                result:
                  summary: 'An result object.'
                  value: { success: true }
                boolz:
                  summary: 'A boolean value.'
                  value: 'false'
```
this error is reported:
```shell
oas3-valid-oas-content-example  `value` property type should be object  paths./users.post.responses[200].content.application/json.examples.boolz.value
```

The specs (3.0.3 and 3.1.0) the type of `value` as `Any`.

`swagger-cli` does not report this as an issue.

spectral version: 5.9.1
OS: macos

**Comment by felipeloha** [comment]
+1

any idea when this will fixed?

**Comment by mnaumanali94** [comment]
@felipeloha changing `value: 'false'` to `value: false` should fix the reported error. The message certainly can be improved though.
